library/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required (VERSION 3.5)
+project (library)
+
+set (CMAKE_CXX_STANDARD 11)
+
+include_directories(${vw_INCLUDE_DIRS}) 
+
+add_executable(library_example library_example.cc)
+target_link_libraries(library_example PRIVATE vw)
+
+add_executable(ezexample_train ezexample_train.cc)
```

Comment:
>ezexample_train [](start = 15, length = 15)

consider moving to example folder

Reply:
I'd like to keep the current structure to avoid changing too much

---
In reply to: [192516514](https://github.com/JohnLangford/vowpal_wabbit/pull/1492#discussion_r192516514) [](ancestors = 192516514)